### ISSUE TITLE
Customers report: table displays country code but filter uses country names

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/wc-admin#1272]
Not a big deal, but a usability improvement we could consider.

Imagine I have customers from Spain (_ES_) and Andorra (_AD_):
![image](https://user-images.githubusercontent.com/3616980/50962462-9824ab00-14ca-11e9-963b-e5d3bb8011ca.png)

I want to filter Spanish customers. After seeing _Spain_ is represented as _ES_ in the table, I expect the country filter will work the same, so I write _ES_...

![image](https://user-images.githubusercontent.com/3616980/50962625-f18cda00-14ca-11e9-9713-8854f491cc1e.png)

... and none of the suggestions are Spain.

Same happens for many other countries. If I didn't know _ES_ represents _Spain_, I would have no way to filter those users without searching for a list of country codes in Google.

***Suggested solutions***
- Customers country filter should suggest options not only based on the name but also the country code.
- A nice addition would be adding a `title` to the country code with the country name, so hovering with the mouse would reveal which country belongs to that code:
![image](https://user-images.githubusercontent.com/3616980/50962987-c6ef5100-14cb-11e9-9a01-cd8af75da96c.png)
That country name should be used also as the screen reader text.
